spaces/domains.json

```
@@ -745,5 +745,6 @@
   "vote.cortexdao.io": "cortexdao.eth",
   "gov.emdx.io": "emdx.eth",
   "vote.evoverses.com": "evoverses.games",
-  "vote.nation3.org": "nation3.eth"
+  "vote.nation3.org": "nation3.eth",
+  "vote.famdao.org":"fam-dao.eth"
```

Please add a space here.
```suggestion
  "vote.famdao.org": "fam-dao.eth"
```